src/features/currentTodo.ts

```diff
@@ -1,19 +1,19 @@
 import { Todo } from '../types/Todo';
 
 // we use string literal as a type to avoid mistype in future
-type RemoveTodoAction = { type: 'currentTodo/REMOVE' };
+type RemoveTodoAction = { type: 'currentTodo/remove' };
```

Comment:
Can use consts for all types.
For example const REMOVE_TODO = 'currentTodo/remove'.

And use it from one place.